2022/24xxx/CVE-2022-24825.json

```diff
@@ -1,18 +1,88 @@
+                                "product_name": "smokescreen",
+                                "version": {
+                                    "version_data": [
+                                        {
+                                            "version_value": "< 0.0.3"
+                                        }
+                                    ]
+                                }
+                            }
+                        ]
+                    },
+                    "vendor_name": "stripe"
+                }
+            ]
+        }
+    },
+    "data_format": "MITRE",
+    "data_type": "CVE",
+    "data_version": "4.0",
     "description": {
         "description_data": [
             {
                 "lang": "eng",
-                "value": "** RESERVED ** This candidate has been reserved by an organization or individual that will use it when announcing a new security problem. When the candidate has been publicized, the details for this candidate will be provided."
+                "value": "Smokescreen is a simple HTTP proxy that fogs over naughty URLs. The primary use case for Smokescreen is to prevent server-side request forgery (SSRF) attacks in which external attackers leverage the behavior of applications to connect to or scan internal infrastructure. Smokescreen also offers an option to deny access to additional (e.g., external) URLs by way of a deny list. There was an issue in Smokescreen that made it possible to bypass the deny list feature by appending a dot to the end of user-supplied URLs, or by providing input in a different letter case. Recommended to upgrade Smokescreen to version 0.0.3 or later."
+            }
+        ]
+    },
+    "impact": {
+        "cvss": {
+            "attackComplexity": "LOW",
+            "attackVector": "NETWORK",
+            "availabilityImpact": "NONE",
+            "baseScore": 5.8,
+            "baseSeverity": "MEDIUM",
+            "confidentialityImpact": "LOW",
+            "integrityImpact": "NONE",
+            "privilegesRequired": "NONE",
+            "scope": "CHANGED",
+            "userInteraction": "NONE",
+            "vectorString": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:C/C:L/I:N/A:N",
+            "version": "3.1"
+        }
+    },
+    "problemtype": {
+        "problemtype_data": [
+            {
+                "description": [
+                    {
+                        "lang": "eng",
+                        "value": "CWE-918: Server-Side Request Forgery (SSRF)"
+                    }
+                ]
             }
         ]
+    },
+    "references": {
+        "reference_data": [
+            {
+                "name": "https://github.com/stripe/smokescreen/security/advisories/GHSA-gcj7-j438-hjj2",
+                "refsource": "CONFIRM",
+                "url": "https://github.com/stripe/smokescreen/security/advisories/GHSA-gcj7-j438-hjj2"
+            },
+            {
+                "name": "github.com/stripe/smokescreen",
```

Comment:
Hey @CallmeMari,

Looks like the "name" and "url" attributes both are missing  "https://". Once that is fixed the process will continue and merge the PR. 

Thanks,
CVE Content Team

Reply:
Thank you for catching this, it has been fixed.